c_api/netutil_c_api.go

```diff
@@ -33,6 +33,15 @@ struct NetworkStatus {
 struct NetworkStatusResponse {
 	struct NetworkStatus Status[NETUTIL_NUM_NETWORKSTATUS];
 };
+
+struct NetworkInterface {
+	char*	Type;
+	char*	ID;
```

Comment:
Should this be PCIAddress? FYI - I will probably move this into a substructure later, but fine where it is for now.

Reply:
I was thinking for userspace CNI, this is not a PCI address, but a socket file path which is unique to vhost interface.
Want to cover both `pci` and `vhost` with one field, open to any suggestion on the field name.

Reply:
Just as an example, I was thinking:
struct SriovData {
	char*	PCIAddress;
}
struct VhostData {
	char*	SocketFile;
	bool		Master;
}
struct NetworkInterface {
	char*	Name;
	char*	Type;
	struct SriovData	Sriov;
	struct VhostData	Vhost;
}

Reply:
This looks more clean with SR-IOV and Vhost being in separate structure, in this case, the `Type` field I guess is the interface type such as `pci` or `vhost`,  once user queries full list of mixed vhost and pci interfaces, he will be able to know from the `type` field which sub-structure to get the data.